lib/main.dart

```diff
@@ -1,115 +1,113 @@
+        ),
+        leading: Text('Versão beta'),
+        actions: <Widget>[Icon(Icons.menu)],
       ),
-      body: Center(
-        // Center is a layout widget. It takes a single child and positions it
-        // in the middle of the parent.
-        child: Column(
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Invoke "debug painting" (press "p" in the console, choose the
-          // "Toggle Debug Paint" action from the Flutter Inspector in Android
-          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
-          // to see the wireframe for each widget.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
+      floatingActionButton: FloatingActionButton(
+        onPressed: () => add(),
+        child: Icon(Icons.add),
+        backgroundColor: Color.fromRGBO(79, 62, 13, 0.932),
+      ),
+      body: ListView.builder(
+        itemCount: items.length,
+        itemBuilder: (BuildContext ctxt, int index) {
+          final item = items[index];
+          return Dismissible(
+            child: CheckboxListTile(
+              title: Text(item.title.toString()),
+              value: item.done,
+              onChanged: (value) {
+                setState(() {
+                  // função do stf que muda o estado dos parametros passados, serve para quando vamos atualizar uma posição
+                  item.done = value;
+                });
+              },
             ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
+            key: Key(item.title.toString()),
+            background: Container(
+              color: Colors.red.withOpacity(0.2),
+              child: Text("Excluir"),
             ),
-          ],
-        ),
+            onDismissed: (direction) {
+              print(direction);
+              remove(index);
+            },
+          );
+        },
       ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: const Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
     );
   }
 }
+//  Container(
+//   child: Center(
+//     child: Text(
+//       'ola mundo',
+//     ),
+//   ),
+// ),
```

Comment:
Please remove unnused comments from the code.

Reply:
Done